docs/metrics/diarization-error.md

```diff
@@ -0,0 +1,103 @@
+# Diarization Error Rate
+Diarization Error Rate (DER) is an important metric used to evaluate speaker diarization systems. It quantifies the overall performance of a speaker diarization system by measuring the ratio of the duration of errors to the total reference speech duration. DER is calculated as a score between 0 and 1, with 0 being a perfect match and 1 being a complete mismatch between candidate and reference diarization.
```

Comment:
Hmm given that the metric is divided by the reference diarization, do you think it can be > 1?